frontend/components/LogoLink/LogoLink.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Link from 'next/link'
+
+const LogoLink = props => {
+  let icon
+
+  switch (props.service) {
+    case 'email':
+      icon = 'far fa-envelope'
+      break
+    case 'twitter':
+      icon = 'fab fa-twitter'
+      break
+    case 'facebook':
+      icon = 'fab fa-facebook'
+      break
+    default:
+      icon = 'fas fa-question'
+  }
```

Comment:
You could just write like this `const icon = fa fa-${props.service}`

Reply:
The problem is that the brand icons and regular icons are under different classes (`fab` and `fas` respectively) and using `fa` result in an empty rectangle.